File: Codechef/Java/Beginner/SmallFactorials.java
import java.util.Scanner;

public class SmallFactorials {
	public static void main(String[] args) {
		Scanner scan = new Scanner(System.in);
		int numberOfElements =  scan.nextInt();
		while(numberOfElements > 0) {
			int number = scan.nextInt();
			int fact = 1;
			for(int i = 1; i <= number; i++) {
				fact *= i;
			}
			System.out.println(fact);
			numberOfElements--;
		}
	}
}

File: Codechef/Python/SmallFactorials.py
numberOfTest = int(input())
while(numberOfTest > 0):
	fact = 1
	number = int(input())
	if(number <= 0):
		print(fact)
	else:
		while(number > 0):
			fact *= number
			number -= 1
		print(fact)
	numberOfTest -= 1
